docs/develop/getting-started/faq.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 4
+---
+
+# Frequently Asked Questions 
+
+This FAQ page is designed to address the most common technical inquiries about WasmEdge. If your question is not directly answered here, please refer to the comprehensive WasmEdge documentation or engage with the active WasmEdge community.
+
+## 1. How does WasmEdge handle memory sharing between modules?
+
+WasmEdge follows the WebAssembly specification, which currently does not support shared memory between different modules. Each module has its own linear memory space.
+
+## 2. Can WasmEdge support model training?
+
+Currently, WasmEdge supports model inference. It uses the WASI-NN API to make predictions using pre-trained models. However, model training is not yet supported.
```

Comment:
Link an article for AI/LLM inference.

Reply:
Could you please provide the link of that article? I am confused which one should i add.

Reply:
https://www.secondstate.io/articles/fast-llm-inference/